### DISCUSSION TITLE
Kade Engine crashes whenever I compile

### DISCUSSION THREAD

**Discussion by Spookandscare** [KadeArchive/Kade-Engine#2497] [Q&A]
With the latest release, I tried compiling it, but whenever it finishes loading it just crashes, it doesn't give me any reason or warning, it just crashes, any help is appreciated!

**Comment by bwkam** [comment]
Alright so try that,

1. [Download this (not a virus)](https://drive.google.com/u/0/uc?id=1JT3EE7dJk9MKNzr2-YbojiMwJ23Mh4L2&export=download), it will automatically open the command prompt and will download all required libraries, it could be a problem of  a missing library.


2. Run these commands in the command prompt

haxelib remove polymod

haxelib git polymod 

paste that to the git path: https://github.com/MasterEric/polymod

Compile the game again and see if it still makes the same thing, hope that helps.
